### PR TITLE
BL-772 Update filters and facets for responsive views

### PR DESCRIPTION
**Pull request by sensei100**

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -146,4 +146,8 @@ $(document).on('turbolinks:load', function() {
  	if ($("div.navbar-form").length == 0) {
  		$("#search-navbar").css("padding-left", "15%");
  	}
+
+	$('#facet-filter-icon').click( function(){
+    $(this).find('span#facet-icons').toggleClass('open-facet-icon').toggleClass('remove-facet-icon');
```
Comment (dkinzer) [inline app/assets/javascripts/application.js]: @sensei100 quick question on this, why is this needed? How can I test it?  Thanks!

Comment (sensei100) [reply app/assets/javascripts/application.js]: @dkinzer They want a different icon to be displayed when the facets are open/closed in the responsive views.  You can see it by using the responsive dev tools in chrome and looking at the yellow icon for the facets.  It should be the filter icon when facets are closed, and show the close icon when they are open.

Comment (dkinzer) [reply app/assets/javascripts/application.js]: Thanks! 👍
